src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.spec.ts

```diff
@@ -93,19 +104,38 @@ describe('DotEditPageNavComponent', () => {
             const activeItem = fixture.debugElement.query(By.css('.edit-page-nav__item--active'));
             expect(activeItem.nativeElement.innerText).toContain('Content');
         });
+
+        it('should update menu items when new PageState', () => {
```

Comment:
This test and ```'should have basic menu items'``` should be in one describe, looks like we are repeating code thats should be in a ```beforeEach```